test/integration/tests/duplicate.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
```

Comment:
Missing Licensing header

Reply:
Assigned to Intel, 2019

Reply:
your free to put the copyright in your name, you're the author.